tests/e2e/restic_helpers.go

```diff
@@ -140,19 +186,19 @@ func disableRestic(namespace string, instanceName string) error {
 func isResticDaemonsetDeleted(namespace string, instanceName string, resticName string) wait.ConditionFunc {
 	err := disableRestic(namespace, instanceName)
 	if err != nil {
-		return nil
+		panic(err)
 	}
 	return func() (bool, error) {
 		config := getKubeConfig()
-		client, err := dynamic.NewForConfig(config)
+		client, err := kubernetes.NewForConfig(config)
 		if err != nil {
 			return false, nil
 		}
-		veleroClient, err := createVeleroClient(client, namespace)
-		if err != nil {
+		// get daemonset in oadp-operator-e2e ns with specified field selector
+		_, errs := client.AppsV1().DaemonSets(namespace).Get(context.TODO(), "restic", metav1.GetOptions{})
```

Comment:
resticName is passed as parameter here. Would it be possible to use that here as well?

Reply:
+1